lib/cli/run-helpers.js

```diff
@@ -83,33 +84,21 @@ exports.list = str =>
  * @private
  */
 exports.handleRequires = async (requires = []) => {
-  const acc = [];
-  for (const mod of requires) {
+  const pluginLoader = PluginLoader.create();
+  for await (const mod of requires) {
```

Comment:
this may not be strictly necessary, but I think it's going to help reason about the order in which requires are loaded.